### ISSUE TITLE
2020-02 review of lesson text, general notes

### ISSUE THREAD

**Issue by mikerenfro** [carpentries-incubator/hpc-intro#297]
Request for comments:

1. "Utilize" could be "use" throughout.
2. Decide on nomenclature for anti-abreviation of HPC. Do we hyphenate high-performance (probably not)? Do we leave it all lowercase?
3. Sometimes we emphasize terms with italics, sometimes with boldface. "The cloud" on 11-hpc-intro is bold, and "The cloud" on 12-cluster is italics. Any significance to the difference? Any preference on just using one?
4. Should images be linked as "../fig/" instead of "fig/" ?
5. Consistent use of "log in to" versus "log into", "log on to", "logon to", "login to"?
6. Check notation for schedulers like "Slurm" instead of "SLURM".
7. Capitalize Environment Modules throughout. Other uses of "module" can probably be lowercase.
8. Some `sbatch` lines have two spaces between the command and the job script name.
9. Decide on capitalization rules for headings. I've title-cased everything in 16-parallel, but I don't think that matches the rest of the episodes.
10. Decide if we want to use the term "login node" throughout? We're definitely logging into something that functions as a login node, even if it also functions as a head node in some cases.
11. Consistently format i.e., and e.g., either with or without italics.
12. Verify accessibility of colors used in figures versus common colorblindness types.

**Comment by wirawan0** [comment]
Commenting on one: "**Decide on capitalization rules for headings**" -- I would agree with the Carpentries style guide: let's do Title Case Capitalization for all headings. I would mean all levels for headings--not just levels 1 and 2 alone.

**Comment by mikerenfro** [comment]
> Consistent use of "log in to" versus "log into", "log on to", "logon to", "login to"?

[Google Ngram](https://books.google.com/ngrams/graph?content=log+into%2Clogin+to%2Clog+in+to%2Clogon+to%2Clog+on+to%2Clog+onto&year_start=1980&year_end=2019&corpus=26&smoothing=3) and [Grammarist](https://grammarist.com/usage/log-on-log-in/) lead me toward "log in to" as the correct one for what we're doing (authenticating to get access to a service).

**Comment by mikerenfro** [comment]
True, and though [Ngram](https://books.google.com/ngrams/graph?content=high-performance+computing%2Chigh+performance+computing&year_start=1980&year_end=2019&corpus=26&smoothing=3#) says the hyphenated version is more common, I'm not finding corroborating evidence on a general Google search for ["high-performance computing"](https://www.google.com/search?q="high-performance"+computing) -- it's a bit over a 2:1 ratio of "high performance" versus "high-performance" on the first page of results.

Even some of the ones using the hyphenated form aren't consistent, and use the non-hyphenated version elsewhere on their pages.

**Comment by tkphd** [comment]
On the matter of *italics* vs. **boldface** for emphasis, I prefer *italics* for readability. To my eye, bold text draws attention; it is more useful when a reader is expected to scan the document for key clauses. Italicized words & phrases blend into the surrounding text better, producing less eye strain when reading normally. (With bold text, my eye keeps jumping from the part I'm trying to read over to the bold word(s).)

Compare the following example text ([source](https://www.thoughtco.com/emphasis-speech-and-composition-1690646)), with keyword in bold:

> **Emphasis** may also be secured by (1) **repetition**; (2) by the **development** of important ideas through supplying plenty of detail; (3) by the **allotment** of more space to the more important ideas; (4) by **contrast**, which focuses the reader's attention; (5) by **selection** of details so chosen that subjects related to the main idea are included and irrelevant material excluded; (6) by **climactic arrangement**; and (7) by **mechanical devices** such as capitalization, **italics**, symbols, and different colors of ink."

or with keywords italicized:

> *Emphasis* may also be secured by (1) *repetition*; (2) by the *development* of important ideas through supplying plenty of detail; (3) by the *allotment* of more space to the more important ideas; (4) by *contrast*, which focuses the reader's attention; (5) by *selection* of details so chosen that subjects related to the main idea are included and irrelevant material excluded; (6) by *climactic arrangement*; and (7) by *mechanical devices* such as capitalization, *italics*, symbols, and different colors of ink."

**Comment by VivaswatS** [comment]
2. As a user and learner, I prefer the term "login node" in the lessons since this is how administrators at my universities have referred to it. This also decreases the cognitive load on the learner by asking them to only remember the one term. **+1** for keeping it as is! 

3. Using this [Toptal Color Blind Filter](https://www.toptal.com/designers/colorfilter/) website, I find that deuteranopia and protanopia cause for red and brown (90% and 99% parallel workload) to look the same on the figures in [Running a parallel job](https://carpentries-incubator.github.io/hpc-intro/16-parallel/). A potential sequential palette could be [this](https://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=5), with more options available on the same website.

**Comment by longr** [comment]
> Sometimes we emphasize terms with italics, sometimes with boldface. "The cloud" on 11-hpc-intro is bold, and "The cloud" on 12-cluster is italics. Any significance to the difference? Any preference on just using one?

> On the matter of _italics_ vs. **boldface** for emphasis, I prefer _italics_ for readability. To my eye, bold text draws attention; it is more useful when a reader is expected to scan the document for key clauses. Italicized words & phrases blend into the surrounding text better, producing less eye strain when reading normally. (With bold text, my eye keeps jumping from the part I'm trying to read over to the bold word(s).)

I agree with this (or I did).   I have looked through the carpentries style guide and cannot see any information on this. Short of guidance, we have two competing issues here: 1) Bold draws the eyes and I suspect this makes it hard to read for neuro-diverse individuals. 2) At the same time italics should be avoided as it makes the text hard to read, especially for those with dyslexia. 

I think the issue actually is not one of whether we should be using bold or italics, it is an issue of why are we using them? I would suggest we do the following:
1) Where we want to highlight a particular concept or make it easier for a learner to find a key concept we use bold (but sparingly - the opening bullets of [Why use a Cluster?](https://carpentries-incubator.github.io/hpc-intro/11-hpc-intro/index.html) is, I believe,  a case of over use).
2) Italics should not be used as they are hard to read (keep them for mathematical variables but consider replacing those with inline code instead)
3) Any instances of Bold and/or italics where we are wanting to highlight a key term to the learner should be replaced with a link to the glossary and the appropriate entry made.  This will allow key terms to be highlighted in a non-intrusive manner, and provide the learner with a quick link to the definition if they are not sure of it.

I am happy to prepare this as a pull request.
